package.json

```diff
@@ -4,7 +4,9 @@
   "private": true,
   "description": "Mono repository for Graphback project",
   "main": "index.js",
-  "dependencies": {},
+  "dependencies": {
+    "@graphql-inspector/core": "^1.26.0"
```

Comment:
Wrong place for this. We should have it in graphback (for the moment).
Remove package-log.json commit (maybe we can add it to git ignore.